Userland/Utilities/image.cpp

```diff
@@ -100,38 +107,36 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     if (!decoder)
         return Error::from_string_view("Failed to decode input file"sv);
 
-    auto frame = TRY(decoder->frame(frame_index)).image;
+    LoadedImage image = { TRY(decoder->frame(frame_index)).image, TRY(decoder->icc_data()) };
 
     if (move_alpha_to_rgb)
-        TRY(do_move_alpha_to_rgb(frame));
+        TRY(do_move_alpha_to_rgb(image));
```

Comment:
FYI: Passing `image.bitmap` also works here

Reply:
In this commit, yes. But a later commit makes the function error out for cmyk images.